apps/interactiveevents/templates/a4_candy_interactive_events/module_detail.html

```diff
@@ -33,7 +33,26 @@
                 <span class="label">{% trans 'Interactive' %}</span>
                 <h1>{{ module.name }}</h1>
                 <div class="mb-2">
-                    {{ module.description | transform_collapsibles | richtext }}
+                    {% if module.description|length > 180 %}
+                        <div id="collapsibleContentEvent" class="collapse show u-no-transition">
+                            {{ module.description|truncatechars:160|richtext }}
+                        </div>
```

Comment:
you use 180 and the 160 on collapsible elements, doens't that mean that 20 are cut off?

Reply:
so if description is bigger than 180, it will show 160 + read more
if description is smaller than 180, it will show full length.

so in case of 181 or 212, it will show 160 + read more
trying to avoid if someone uses 181 chars, then behind read more is only one letter.

or am I wrong? yeah its a bit of knot in the head..